### PR TITLE
doc change

### PR DESCRIPTION
**Pull request by uday610**
1) added Developer's section by reorganize topic
2) modified tool section heading
3) minor 1 line change in xma_user_guide.rst

### CODE REVIEW

File: src/runtime_src/doc/toc/tools.rst [hunk 1]
```diff
@@ -1,7 +1,4 @@
-Board Tools and Utilities
--------------------------
-
 xbutil
-~~~~~~
+-------
```
Comment (sonals) [inline src/runtime_src/doc/toc/tools.rst]: Remove one extra '-'

File: src/runtime_src/doc/toc/xclbintools.rst [hunk 2]
```diff
@@ -1,5 +1,2 @@
-Xclbin Tools
+xclbinutil
 ------------
```
Comment (sonals) [inline src/runtime_src/doc/toc/xclbintools.rst]: Remove 2 extra '-'

Comment (uday610) [reply src/runtime_src/doc/toc/xclbintools.rst]: ok I removed those

File: src/runtime_src/doc/toc/tools.rst (hunk 1 above)
Comment (uday610) [reply src/runtime_src/doc/toc/tools.rst]: ok I removed those

**Pull request merged**